MSMAll/scripts/SingleSubjectConcat.sh

```diff
@@ -87,10 +87,12 @@ do
 	runFrames[index]="$NumFrames"
 	TR=`${Caret7_Command} -file-information "${FullDenseTCS}" -only-step-interval`
 	runTR[index]="$TR"
-	if [ $((index)) -eq 0 ]; then
+	if ((index -eq 0 )); then
```

Comment:
`-eq` is not valid in `(( ))` (it gets interpreted as "minus variable `eq`"), use `==`.  The main reason for switching to `(( ))` is that `==` (and needing fewer `$`) is more readable in my opinion.

Reply:
fixed now